pkg/services/ngalert/api/generated_base_api_prometheus.go

```diff
@@ -12,30 +12,30 @@ import (
 	"github.com/grafana/grafana/pkg/api/response"
 	"github.com/grafana/grafana/pkg/api/routing"
 	"github.com/grafana/grafana/pkg/middleware"
-	"github.com/grafana/grafana/pkg/models"
+	contextmodel "github.com/grafana/grafana/pkg/services/contexthandler/model"
```

Comment:
these files are autogenerated by running:
`cd pkg/services/ngalert/api/tooling && make gen`

based on [these](https://github.com/grafana/grafana/tree/main/pkg/services/ngalert/api/tooling/swagger-codegen/templates) template files; therefore, they have to be updated appropriately as well

Reply:
thank you, didn't notice that this is a generated file